src/System.Private.CoreLib/src/System/Globalization/CultureInfo.CoreRT.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.Serialization;
+using System.Threading;
+
+#if ENABLE_WINRT
+using Internal.Runtime.Augments;
+#endif
+
+namespace System.Globalization
+{
+    public partial class CultureInfo : IFormatProvider, ICloneable
+    {
+        [ThreadStatic]
+        private static CultureInfo s_currentThreadCulture;
+        [ThreadStatic]
+        private static CultureInfo s_currentThreadUICulture;
+
+        ////////////////////////////////////////////////////////////////////////
+        //
+        //  CurrentCulture
+        //
+        //  This instance provides methods based on the current user settings.
+        //  These settings are volatile and may change over the lifetime of the
+        //  thread.
+        //
+        ////////////////////////////////////////////////////////////////////////
+
+        //
+        // We use the following order to return CurrentCulture and CurrentUICulture
+        //      o   Use WinRT to return the current user profile language
+        //      o   use current thread culture if the user already set one using CurrentCulture/CurrentUICulture
+        //      o   use thread culture if the user already set one using DefaultThreadCurrentCulture
+        //          or DefaultThreadCurrentUICulture
+        //      o   Use NLS default user culture
+        //      o   Use NLS default system culture
+        //      o   Use Invariant culture
+        //
+        public static CultureInfo CurrentCulture
+        {
+            get
+            {
+                CultureInfo ci = GetUserDefaultCultureCacheOverride();
```

Comment:
It would be for both getter and setter to use the same style. Either use ifdef in both or use the `Override` method in both. (I would be fine with the ifdef here.)